common/fs/src/cache/mod.rs

```diff
@@ -299,6 +301,8 @@ impl FileSystem {
             }
         });
 
+        let fs_start_time = SystemTime::now();
```

Comment:
not monotonic

Reply:
i'm not sure what you're getting at - but this is just the system time that is captured to compare when a file was created for tail